cabal-plan.cabal

```diff
@@ -62,6 +62,7 @@ executable cabal-plan
     -- dependencies w/ inherited version ranges via 'cabal-plan' library
     build-depends: cabal-plan
                  , base
+                 , algebraic-graphs  == 0.0.5
```

Comment:
1. Why the odd version constraint? Does `algebraic-graphs` fail to follow the PVP?
2. as the comment stats, this group of build-depends is for those whose constraints are inherited from the library part; there's a second group below for other deps.

Reply:
>  Does `algebraic-graphs` fail to follow the PVP?

@hvr From the library description: `This is an experimental library and the API will be unstable until version 1.0.0`. However, it looks like I have to reconsider this given that Alga already has several users. So I can start following the PVP from the next release if need be.

Reply:
@snowleopard Well, being experimental and following the PVP is not a contradicition (it just means you'll have to make sure to major bump more frequently; long time ago, we had a stabilify-flag in the cabal file for that, but it was considered pointless. When uploading to Hackage, we expect you to follow the PVP; if your package is too experimental for other packages start  depending upon it, it doesn't belong on Hackage yet; see also http://hackage.haskell.org/upload ; you touched a topic that's  quite a pet peeve of mine as it's likely the one area of the Haskell ecosystem I've invested most time and effort in... :-)

Reply:
@hvr Agreed. The next release version will be `0.1.0` and Alga will comply with PVP.